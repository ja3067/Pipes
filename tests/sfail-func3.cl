x : str = "hello"

def foo(arr : str):
	arr = 3.0
	return arr
