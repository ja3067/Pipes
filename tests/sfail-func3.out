STypeError: expression of type float cannot be assigned to variable 'arr' with explicit type str
